credentialbuilder/src/main/java/org/apache/ranger/credentialapi/CredentialReader.java

```diff
@@ -17,80 +17,82 @@
-		  alias=alias.trim();
-		  if(CrendentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixJceks) ||
-				  CrendentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixLocalJceks) ||
-				  CrendentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixBcfks) ||
-				  CrendentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixLocalBcfks)){
-			  conf.set(CredentialProviderFactory.CREDENTIAL_PROVIDER_PATH,
-					   //UserProvider.SCHEME_NAME + ":///," +
-			  CrendentialProviderPath);
-		  }else{
-			  if(CrendentialProviderPath.startsWith("/")){
-			  	 if(StringUtils.equalsIgnoreCase(storeType, "bcfks")) {
-					 conf.set(CredentialProviderFactory.CREDENTIAL_PROVIDER_PATH, CrendentialProviderPath);
-				 } else {
-					 conf.set(CredentialProviderFactory.CREDENTIAL_PROVIDER_PATH,
-							 //UserProvider.SCHEME_NAME + ":///," +
-							 JavaKeyStoreProvider.SCHEME_NAME + "://file" + CrendentialProviderPath);
-				 }
+            String crendentialProviderPrefixBcfks = "bcfks" + "://file";
+            String crendentialProviderPrefixLocalBcfks = "localbcfks" + "://file";
+            crendentialProviderPrefixBcfks = crendentialProviderPrefixBcfks.toLowerCase();
+            crendentialProviderPrefixLocalBcfks = crendentialProviderPrefixLocalBcfks.toLowerCase();
 
-			  }else{
-				  conf.set(CredentialProviderFactory.CREDENTIAL_PROVIDER_PATH,
-						   //UserProvider.SCHEME_NAME + ":///," +
-						  JavaKeyStoreProvider.SCHEME_NAME + "://file/" + CrendentialProviderPath);
-			  }
-		  }
-		  List<CredentialProvider> providers = CredentialProviderFactory.getProviders(conf);
-		  List<String> aliasesList=new ArrayList<String>();
-		  CredentialProvider.CredentialEntry credEntry=null;
-		  for(CredentialProvider provider: providers) {
-              //System.out.println("Credential Provider :" + provider);
-			  aliasesList=provider.getAliases();
-			  if(aliasesList!=null && aliasesList.contains(alias.toLowerCase())){
-				  credEntry=null;
-				  credEntry= provider.getCredentialEntry(alias.toLowerCase());
-				  pass = credEntry.getCredential();
-				  if(pass!=null && pass.length>0){
-					  credential=String.valueOf(pass);
-					  break;
-				  }				
-			  }
-		  }
-	  }catch(Exception ex){
-		  ex.printStackTrace();
-		  credential=null;
-	  }
-	  return credential;
-  }
+            credentialProviderPath = credentialProviderPath.trim();
+            alias = alias.trim();
+            if (credentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixJceks) ||
+                    credentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixLocalJceks) ||
+                    credentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixBcfks) ||
+                    credentialProviderPath.toLowerCase().startsWith(crendentialProviderPrefixLocalBcfks)) {
+                conf.set(CredentialProviderFactory.CREDENTIAL_PROVIDER_PATH,
```

Comment:
Please fold unnecessary splits into a single line, here and other places; this is a major code readability issue in several modules.

Reply:
Done